plugins/dotnet-auth-core-identity/src/core/create-static-file-map.ts

```diff
@@ -7,20 +7,24 @@ export async function createStaticFileFileMap(
   destPath: string,
   filePath: string,
   context: dotnetTypes.DsgContext,
-  classReference?: ClassReference
+  classReferences?: ClassReference[]
 ): Promise<FileMap<CodeBlock>> {
   const fileMap = new FileMap<CodeBlock>(context.logger);
 
   if (!context.resourceInfo) return fileMap;
   const resourceName = pascalCase(context.resourceInfo.name);
   let fileContent = await readFile(filePath, "utf-8");
   fileContent = fileContent.replace("ServiceName", resourceName);
+  fileContent = fileContent.replace(
+    "ServiceNameDbContext",
+    `${resourceName}DbContext`
+  );
 
   const file: IFile<CodeBlock> = {
     path: destPath,
     code: new CodeBlock({
       code: fileContent,
-      references: classReference && [classReference],
+      references: classReferences && classReferences,
```

Comment:
is this on purpose? should not just be :
```suggestion
      references: classReferences,
```

Reply:
@overbit  Yep, This is an optional param, and the type is an array of classReferences, So I don't need the "[]" 
<img width="473" alt="image" src="https://github.com/amplication/plugins/assets/97830649/c117a7fd-b443-45ec-a9bd-bfb7686c811b">